### ISSUE TITLE
Sidebar is too large & not collapsible

### ISSUE THREAD

**Issue by ericclemmons** [statelyai/xstate-viz#381]
**Actual Result**

> ![Screen Shot 2022-08-19 at 10 22 13 AM](https://user-images.githubusercontent.com/15182/185652507-5888def8-3e79-4c0d-9317-08648ffb5449.png)

**Expected Result**

- [ ] Collapsed by default
- [ ] Collapsible like stately editor
- [ ] (Or re-use stately editor 🙏)

Something like this: 
![Screen Shot 2022-08-19 at 10 25 34 AM](https://user-images.githubusercontent.com/15182/185653150-e2e8b3e4-2eeb-40a6-a957-ea63468d63c9.png)

**Reproduction**

Using https://github.com/simeonc/storybook-xstate-addon

**Additional context**

**Comment by davidkpiano** [comment]
Good catch; if possible, try pointing the Stately link to `https://stately.ai/viz/embed?mode=viz&inspect`, which should hide that sidebar.
